updatecli/updatecli.d/update-deps.yaml

```diff
@@ -0,0 +1,168 @@
+    kind: dockerimage
+    sourceid: allowPrivilegeEscalationPolicyTag
+    spec:
+      image: ghcr.io/kubewarden/policies/allow-privilege-escalation-psp
+  isHostNamespacePolicyCurrent:
+    name: Is there a new host-namespaces-psp tag
+    kind: dockerimage
+    sourceid: hostNamespacePolicyTag
+    spec:
+      image: ghcr.io/kubewarden/policies/host-namespaces-psp
+  isPodPrivilegedPolicyCurrent:
+    name: Is there a new pod-privileged tag
+    kind: dockerimage
+    sourceid: podPrivilegedPolicyTag
+    spec:
+      image: ghcr.io/kubewarden/policies/pod-privileged
+  isUserGroupPolicyCurrent:
+    name: Is there a new user-group-psp tag
+    kind: dockerimage
+    sourceid: userGroupPolicyTag
+    spec:
+      image: ghcr.io/kubewarden/policies/user-group-psp
+  isHostPathsPolicyCurrent:
+    name: Is there a new hostpaths-psp tag
+    kind: dockerimage
+    sourceid: hostPathsPolicyTag
+    spec:
+      image: ghcr.io/kubewarden/policies/hostpaths-psp
+  isCapabilitiesPolicyCurrent:
+    name: Is there a new capabilities-psp tag
+    kind: dockerimage
+    sourceid: capabilitiesPolicyTag
+    spec:
+      image: ghcr.io/kubewarden/policies/capabilities-psp
+
+targets:
+  updatekubectlTag:
+    name: Update kubectl image tag
+    kind: yaml
+    sourceid: kubectlImageTag
+    spec:
+      file: "charts/kubewarden-controller/values.yaml"
+      key: "$.preDeleteJob.image.tag"
+  updateAllowPrivilegeEscalationPolicyTag:
+    name: Update allow-privilege-escalation-psp tag
+    kind: yaml
+    sourceid: allowPrivilegeEscalationPolicyTag
+    spec:
+      file: "charts/kubewarden-defaults/values.yaml"
+      key: "$.recommendedPolicies.allowPrivilegeEscalationPolicy.module.tag"
+  updateHostNamespacePolicyTag:
+    name: Update host-namespaces-psp tag
+    kind: yaml
+    sourceid: hostNamespacePolicyTag
+    spec:
+      file: "charts/kubewarden-defaults/values.yaml"
+      key: "$.recommendedPolicies.hostNamespacePolicy.module.tag"
+  updatePodPrivilegedPolicyTag:
+    name: Update pod-privileged tag
+    kind: yaml
+    sourceid: podPrivilegedPolicyTag
+    spec:
+      file: "charts/kubewarden-defaults/values.yaml"
+      key: "$.recommendedPolicies.podPrivilegedPolicy.module.tag"
+  updateUserGroupPolicyTag:
+    name: Update user-group-psp tag
+    kind: yaml
+    sourceid: userGroupPolicyTag
+    spec:
+      file: "charts/kubewarden-defaults/values.yaml"
+      key: "$.recommendedPolicies.userGroupPolicy.module.tag"
+  updateHostPathsPolicyTag:
+    name: Update hostpaths-psp tag
+    kind: yaml
+    sourceid: hostPathsPolicyTag
+    spec:
+      file: "charts/kubewarden-defaults/values.yaml"
+      key: "$.recommendedPolicies.hostPathsPolicy.module.tag"
+  updateCapabilitiesPolicyTag:
+    name: Update capabilities-psp tag
+    kind: yaml
+    sourceid: capabilitiesPolicyTag
+    spec:
+      file: "charts/kubewarden-defaults/values.yaml"
+      key: "$.recommendedPolicies.capabilitiesPolicy.module.tag"
```

Comment:
Where are the tags updated? Reading script I have the impression that script just checks if the tag from the values files are the latest ones. But I do not understand where are the targets that will update it with the latest version. Furthermore, where the latest tags are loaded? I think we should do something similar of  the [dockerimage example](https://www.updatecli.io/docs/plugins/resource/docker_image/)

Reply:
To test the script, I've merge this PR into my helm-charts repository main branch and [downgrade two policies](https://github.com/jvanz/helm-charts/commit/26cf482f09f300a6ba9217f104071279480f27b7). Then, I've ran the [CI](https://github.com/jvanz/helm-charts/actions/runs/8347702614/job/22847918282). The CI is green but no PR is open. In the logs we can see that script ran every step but it does not update the tags.

Reply:
Fixed now with a rebase. One doesn't need conditions, just to set `target.foo.scmid`.

Tested in my fork:
https://github.com/viccuad/helm-charts/actions/runs/8393105846/job/22987348660
https://github.com/viccuad/helm-charts/pull/2